### PR TITLE
Adding documentation for seedimage

### PR DESCRIPTION
**Pull request by frelon**
Replace the manual ISO creation in quickstart with seed-image resource.

Add page for seedimage reference.

### CODE REVIEW

**Comment by frelon** [comment]
@nunix are you also doing changes to the quickstart? just wanted to give a heads up :+1:

**Review by fgiudici — COMMENTED** [review]
Looks good here!
The changes should end up on the "cli" quickstart only of @nunix quickstart rework .
I would anyway wait for @nunix review before merging.

File: docs/quickstart-cli.md [hunk 1]
```diff
@@ -69,12 +71,22 @@ Make sure to modify the registration.yaml above to set the proper install device
 The SD-card on a Raspberry Pi is usually `/dev/mmcblk0`.
 :::
 
+<Tabs>
+<TabItem value="seedImagex86" label="Seed Image (x86_64)" default>
+<CodeBlock language="yaml" title="seedimage.yaml" showLineNumbers>{SeedImage}</CodeBlock>
+</TabItem>
+<TabItem value="seedImagerpi" label="Seed Image for Raspberry Pi" default>
+The seed image is not yet used for Raspberry Pi and will have to be generated manually in the section [Preparing the seed image manually](./quickstart.md#preparing-the-installation-seed-image)
```
Comment (davidcassany) [inline docs/quickstart-cli.md:79]: This URL seams to be broken within the preview

**Review by davidcassany — APPROVED** [review]
LGTM, just small comment on a URL

File: docs/quickstart-cli.md (hunk 1 above)
Comment (frelon) [reply docs/quickstart-cli.md]: Good catch!

**Pull request merged**